### PR TITLE
Move GA webpack entry to the main application entry

### PR DESCRIPTION
**Pull request by sndrs**
_n.b. depends on #7086_

## What does this change?

- move the GA entry code to the main webpack entry
- move the GA setup script out of `bootCmp` and into the GA index file

## Why?

allows webpack to properly dedupe common dependencies and make the code easier to reason about

### CODE REVIEW

**Comment by mxdvl** [comment]
A good refactor that should make it easier to reason about GA in the future. It was always confusing to me why it got some different treatment.

File: dotcom-rendering/src/web/browser/index.ts [hunk 1]
```diff
@@ -3,13 +3,15 @@ import './webpackPublicPath';
...
 import { dynamicImport } from './dynamicImport';
+import { ga } from './ga';
 import { islands } from './islands';
 import { ophan } from './ophan';
 import { sentryLoader } from './sentryLoader';
 import { startup } from './startup';
 
 startup('bootCmp', null, bootCmp);
 startup('ophan', null, ophan);
+startup('ga', null, ga);
```
Comment (arelra) [inline dotcom-rendering/src/web/browser/index.ts:14]: Should `ga` should be lower priority and hence initialised later than islands?

File: dotcom-rendering/package.json [hunk 2]
```diff
@@ -36,12 +36,7 @@
 		"build-storybook": "node --max-old-space-size=4096 $(yarn bin)/build-storybook",
 		"makeBuild": "NODE_ENV=production CI_ENV=github webpack --config ./scripts/webpack/webpack.config.js"
 	},
-	"bundlesize": [
-		{
-			"path": "./dist/ga.*.js",
-			"maxSize": "21 kB"
-		}
-	],
+	"bundlesize": [],
```
Comment (arelra) [inline dotcom-rendering/package.json:39]: This is an aside but I don't think we're using the package `bundlesize` correctly?

Running `make bundlesize` and `yarn bundlesize` locally just returns reports of one random chunk per build:

```
 PASS  ./dist/ga.legacy.6562512b7657bf34b173.js: 3.31KB < maxSize 21KB (gzip) 
 PASS  ./dist/ga.modern.bfb6df33849398fbc36b.js: 2.77KB < maxSize 21KB (gzip) 
 PASS  ./dist/ga.variant.73f43741dc853772cc20.js: 2.82KB < maxSize 21KB (gzip) 
```

If we no longer use or rely on this then we can also remove the dependency and scripts from `makefile` and `ci-dcr.sh`

Comment (arelra) [reply dotcom-rendering/package.json:39]: @OllysCoding or @mxdvl might have more context on this

**Review by arelra — APPROVED** [review]

File: dotcom-rendering/src/web/browser/index.ts (hunk 1 above)
Comment (sndrs) [reply dotcom-rendering/src/web/browser/index.ts:14]: i'm not sure - we run ophan asap so i moved it there. happy to move if it's the wrong place.

these are all async calls anyway, so they effectively run concurrently:

image

GA needs consent before it even requests the library, so it won't be doing much till then:

image

File: dotcom-rendering/package.json (hunk 2 above)
Comment (jamesgorrie) [reply dotcom-rendering/package.json:39]: A little bit of context as to why this was added in the first place:
https://github.com/guardian/dotcom-rendering/pull/727#discussion_r316307199

This does seem to suggest we don't use it at all, but maybe had the intention to.

If that's the case we should just remove it and add it when we know what we want to test with it so we don't need to engineer around it going forward.

**Review by jamesgorrie — APPROVED** [review]
This is such a great simplification in how we load this in!

**Pull request merged**